modules.d/90dmsquash-live-autooverlay/create-overlay.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/sh
+
+type getarg > /dev/null 2>&1 || . /lib/dracut-lib.sh
+
+if getargbool 0 rd.live.debug -n -y rdlivedebug; then
+    exec > /tmp/create-overlay.$$.out
+    exec 2>> /tmp/create-overlay.$$.out
+    set -x
+fi
+
+gatherData() {
+    overlay=$(getarg rd.live.overlay)
+    if [ -z "$overlay" ]; then
+        info "Skipping overlay creation: kernel command line parameter 'rd.live.overlay' is not set"
+        exit 0
+    fi
+    # shellcheck disable=SC2086
```

Comment:
Just wondering - is there any particular reason why is the shellcheck's warning suppressed instead of double-quoting the string (here and in other places as well)?

Reply:
No shell dark magic going on, here. I did it for a couple reasons...
1. Based on the possible contents of this variable, there's no way it could include a space when used with this module. Feel free to add quotes. This is really just symptom of my (probably strange) personal preferences.
2. Consitency with similar lines in the rest of the file. There were other lines in the file that could not be resolved by adding quotes due to nesting and required multiple shellcheck overrides. In those cases, I opted for the more readable single line of code rather than introducing other variables.